test/drop_test/action.py

```diff
@@ -545,9 +552,8 @@ def __init__(self, uuid_slot: int, fid):
         self._fid = fid
 
     async def run(self, drop: ffi.Drop):
-        UUIDS_LOCK.acquire()
-        drop.remove_transfer_file(UUIDS[self._uuid_slot], self._fid)
-        UUIDS_LOCK.release()
+        with UUIDS_LOCK:
```

Comment:
Me like it :+1:

Reply:
That was actually the thing that was blocking me when I mentioned it. A ton of `print()` later I found it :D